docs/source/cpp/examples/dataset_skyhook_scan_example.rst

```diff
@@ -0,0 +1,93 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. default-domain:: cpp
+.. highlight:: cpp
+
+=====================
+Arrow Skyhook example
+=====================
+
+The file ``cpp/examples/arrow/dataset_skyhook_scan_example.cc``
+located inside the source tree contains an example of using Skyhook to 
+offload filters and projections to a Ceph cluster.
+
+Instuctions
+===========
+
+.. note::
+   The instructions below are for ubuntu 20.04 or above.
+
+1. Install Ceph and Skyhook dependencies.
+
+.. code-block:: bash
```

Comment:
I think these need indentation to be properly formatted as part of the list, but I can push a fix later

Reply:
I fixed the indentation. Is it ok now ?

Reply:
See https://github.com/uccross/skyhookdm-arrow/pull/294

Reply:
Done